### DISCUSSION TITLE
Code-server with code-ready environment and Google drive upload, link in description.

### DISCUSSION THREAD

**Discussion by vital987** [coder/code-server#3551] [Show and tell]
Here is the [Link](https://github.com/vital987/code-server), refer readme for more info.

**Comment by jsjoeio** [comment]
This is awesome 👏 What's an example of where you might use the Google Drive upload? Is that for file storage, or something else?

**Reply by vital987** [reply]
When you complete your work, just press **DriveUpload** Button, it will auto-archive your files present in your workspace with a zipfile named with timestamp(if you didn't custom name it) and will save in **CodeServer Saves** directory(you can rename the directory) in your drive. It's made from drive API, Refer project readme for more info😄.

**Reply by nx2k3** [reply]
Hey I just build the docker image from your repo,
It is amazing
custom login screen
you should update README with login screen

**Reply by vital987** [reply]
Added screenshots in readme 😄
